src/shipit_code_coverage/shipit_code_coverage/notifier.py

```diff
@@ -45,8 +45,19 @@ def notify(self):
 
             try:
                 rev = changeset['node']
-                coverage = wait_until(lambda: self.get_coverage_summary(rev), 10)
-                if coverage is None:
+
+                class ResponseAcceptedError(Exception):
+                    pass
+
+                def get_coverage():
+                    summary = self.get_coverage_summary(rev)
+                    if not summary:
+                        raise ResponseAcceptedError()
+                    return summary
+
+                try:
+                    coverage = retry(get_coverage)
```

Comment:
wait_until was not catching the exception before, so we shouldn't catch the exception now.

Reply:
Done in 0385197